### ISSUE TITLE
RangeError: Maximum call stack size exceeded when running scaff

### ISSUE THREAD

**Issue by DaCurse** [galElmalah/scaffolder#18]
Either when running `scaff i` or `scaff create` I get a `RangeError: Maximum call stack size exceeded`.
Call stack:
```
$ scaff i
RangeError: Maximum call stack size exceeded
    at Object.readdirSync (fs.js:974:21)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:19:25)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
    at findTemplate (%appdata%\npm\node_modules\scaffolder-cli\src\commandsBuilder\index.js:32:10)
```

When running `scaff create` I do have a `scaffolder` folder in my working directory and I am specifying a template that exists within it. I haven't tried running other commands.

Node version: `14.5.0`
NPM version: `6.14.5`

**Comment by galElmalah** [comment]
Hey, thanks for reporting this issue!
I'm currently a bit sick :/  got a mild corona case...
I will address this in a few days max 😄

**Comment by galElmalah** [comment]
since I can't reproduce this I have added a small heuristic that won't let the search depth of scaffolder templates exceed 25 levels.
meaning any scaffolder folder that's in that range will be found. this is published under version `1.6.16`.
can you try it and let me know if the issue still persist? 
regardless, I will take a closer look at this once ill get my shit together 😷 🤒 🤧

**Comment by galElmalah** [comment]
@DaCurse this should be fixed now if the issue persists please let me know :)
